plugins/CoreVisualizations/CoreVisualizations.php

```diff
@@ -37,6 +39,21 @@ public function deleteUser($userLogin)
         ViewDataTableManager::clearUserViewDataTableParameters($userLogin);
     }
 
+    public function addViewDataTable(&$viewDataTable)
+    {
+        if (Common::getRequestVar('pivotBy', '')) {
+            $tableToRemove = 'Visualizations\HtmlTable';
+        } else {
+            $tableToRemove = 'HtmlTable\PivotBy';
```

Comment:
Because they are both the same HtmlTable, just the Pivot with some extra logic in case Pivot is enabled. We don't want to use two HtmlTables in the UI. Not sure what there is to explain